nhrpd/nhrp_interface.c

```diff
@@ -167,6 +167,7 @@ static void nhrp_interface_interface_notifier(struct notifier_block *n,
 	case NOTIFY_INTERFACE_CHANGED:
 		nhrp_interface_update_mtu(nifp->ifp, AFI_IP);
```

Comment:
I believe mtu and source commands are called by nhrp_interface_update_nbma as well, is it possible to avoid calling them twice?

Reply:
Thanks for the review! Yes, you are right. We can use mtu and source commands via nhrp_interface_update_nbma. And I can still switch between VRFs without problems.